examples/restful-ws-quarkus/src/main/java/io/cloudevents/examples/quarkus/client/UserEventsGenerator.java

```diff
@@ -31,16 +32,15 @@
     @Inject
     @RestClient
     UserClient userClient;
+    
+    long userCount=0;
 
-    public void init(@Observes StartupEvent startupEvent) {
-        Multi.createFrom().ticks().every(Duration.ofSeconds(2))
-            .onItem()
-            .transform(this::createEvent)
-            .subscribe()
-            .with(event -> {
-                LOGGER.info("try to emit user: {}", event.getId());
-                userClient.emit(event);
-            });
+    @Scheduled(every="2s")
+    public void init() {
+        Uni.createFrom().item(createEvent(userCount++)).subscribe().with( event -> {
+            LOGGER.info("try to emit user: {}", event.getId());
+            userClient.emit(event);
+        });
```

Comment:
Far more clean, I like it

Reply:
I didn't tested this, but is the `Uni<>` needed here? Wouldn't `userClient.emit(createEvent(userCount++))` work as well?

Reply:
Sure that works. I was trying to keep a bit of Mutiny in there, but I guess this is cleaner and examples should be clean. I will update the PR.